api/room/schema.py

```diff
@@ -114,22 +130,27 @@ class Arguments:
         image_url = graphene.String()
         calendar_id = graphene.String()
         cancellation_duration = graphene.Int()
+        room_tags = graphene.List(graphene.Int)
     room = graphene.Field(Room)
 
     @Auth.user_roles('Admin')
     def mutate(self, info, room_id, **kwargs):
         validate_empty_fields(**kwargs)
         query_room = Room.get_query(info)
         active_rooms = query_room.filter(RoomModel.state == "active")
-        exact_room = active_rooms.filter(RoomModel.id == room_id).first()
-        if not exact_room:
+        room = active_rooms.filter(RoomModel.id == room_id).first()
+        if not room:
             raise GraphQLError("Room not found")
 
         admin_roles.update_delete_rooms_create_resource(room_id)
-        update_entity_fields(exact_room, **kwargs)
-
-        exact_room.save()
-        return UpdateRoom(room=exact_room)
+        room_tags = []
+        if kwargs.get('room_tags'):
+            room_tags = kwargs.pop('room_tags')
+        update_entity_fields(room, **kwargs)
+        previous_tags = room.room_tags
+        previous_tags.clear()
```

Comment:
I've tried out `room.room_tags.clear()` and it seems to work as well.
Is there a reason for using a pointer to call .clear()?

Reply:
I guess she is doing that to make it descriptive by indicating that it is the previous tags that are being cleared.

Reply:
Was trying to make it more readable but sure this can be worked on too.